package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "aws-sqs-redriver",
+  "version": "1.0.0",
+  "description": "Re-drives messages from one SQS queue to another, e.g. to re-process \"dead letters\"",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --cache .",
+    "build": "rm -rf lib && babel src --out-dir lib",
```

Comment:
- [ ] Please use `--out-dir dist` or maybe `--out-dir build` if you don't like _dist_